build.sh

```diff
@@ -1,3 +1,7 @@
+#!/bin/bash
+
+# Builds a local docker image
+
```

Comment:
I think `build.sh` is no longer used. It appears that `build-and-push-to-dockerhub.sh` is used instead...but I could be wrong.